public/app/features/dashboard-scene/settings/variables/editors/ConstantVariableEditor.tsx

```diff
@@ -11,9 +15,40 @@ interface ConstantVariableEditorProps {
 export function ConstantVariableEditor({ variable }: ConstantVariableEditorProps) {
   const { value } = variable.useState();
 
-  const onConstantValueChange = (event: React.FormEvent<HTMLInputElement>) => {
+  const onConstantValueChange = (event: FormEvent<HTMLInputElement>) => {
     variable.setState({ value: event.currentTarget.value });
   };
 
-  return <ConstantVariableForm constantValue={String(value)} onChange={onConstantValueChange} />;
+  return <ConstantVariableForm constantValue={value.toString()} onChange={onConstantValueChange} />;
+}
+
+export function getConstantVariableOptions(variable: SceneVariable): OptionsPaneItemDescriptor[] {
+  if (!(variable instanceof ConstantVariable)) {
+    console.warn('getConstantVariableOptions: variable is not a ConstantVariable');
+    return [];
+  }
+
+  return [
+    new OptionsPaneItemDescriptor({
+      title: t('dashboard-scene.constant-variable-form.label-value', 'Value'),
+      render: () => <ConstantFieldPlaceholder variable={variable} />,
+    }),
+  ];
+}
+
+function ConstantFieldPlaceholder({ variable }: { variable: ConstantVariable }) {
```

Comment:
why the Placeholder suffix? (vs ConstantValueInput or something similar)

Reply:
this part seems a bit strange.  can you elaborate on why this is needed?  wasn't sure what to name it.  seemed like a temporary/placeholder component for the actual ConstantVariableForm?